### PR TITLE
Python 3 compatibility fixes

### PR DESCRIPTION
**Pull request by tehasdf**
Just enough of them to make the agent-packager be able to
run in the cloudify-agent CI

### CODE REVIEW

File: agent_packager/cli.py [hunk 1]
```diff
@@ -16,23 +16,17 @@
...
 
+lgr = logger.init()
 
 def ver_check():
-    import pkg_resources
     version = None
-    try:
-        version = pkg_resources.get_distribution('agent_packager').version
-    except Exception as e:
-        print(e)
```
Comment (tehasdf) [inline agent_packager/cli.py]: wtf

**Comment by tehasdf** [comment]
CI will probably be red but it's red on master too, i'm not going to fix it now

File: agent_packager/cli.py (hunk 1 above)
Comment (geokala) [reply agent_packager/cli.py]: Probably 3.1 era compatibility?

Comment (geokala) [reply agent_packager/cli.py]: Blame says the reason is "First commit", from 2014.

**Pull request merged**